src/styles/abstracts/_functions.scss

```diff
@@ -95,3 +95,7 @@ $px-only: false !default;
 @mixin baseline($zoom: 100%) {
```

Comment:
you spilled some mixins in your functions file 😆

Reply:
Ya, these are tied to a function therein. Will separate.